plone/app/tiles/imagescaling.py

```diff
@@ -159,7 +160,7 @@ def scale(self, fieldname=None, scale=None, height=None, width=None, **parameter
             width, height = available[scale]
         storage = AnnotationStorage(self.context, self.modified)
         info = storage.scale(
-            factory=self.create, fieldname=fieldname, height=height, width=width, **parameters
+            factory=self.create, fieldname=fieldname, height=height, width=width, direction=direction, **parameters
```

Comment:
Is it possible to implement new `mode` instead of deprecated `direction` here?

see
https://github.com/plone/plone.scale/blob/60baf9804ec61462028900af1ed3a9d95b063c33/plone/scale/scale.py#L320-L352

Reply:
True. But before that, changes need to be implemented in plone.namedfile. plone.app.tiles come after.

Reply:
Then it's out of scope right now here